### PR TITLE
feat: docker image

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/hirosystems/stacks-scout/issues/22

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+ENV STACKS_NODE_P2P_HOST=seed.mainnet.hiro.so
+ENV STACKS_NODE_P2P_PORT=20444
+
+ENV DATA_STORAGE_DIR=/state
+VOLUME /state
+
+EXPOSE $DATA_PLANE_PORT
+EXPOSE $CONTROL_PLANE_PORT
+EXPOSE $PROMETHEUS_PORT
```
Comment (CharlieC3) [inline Dockerfile:45]: Maybe these can be made as Dockerfile comments with `#`?

Comment (zone117x) [reply Dockerfile:45]: Comments for descriptions of the port purposes, or just commenting them out?

Comment (CharlieC3) [reply Dockerfile:45]: Just commenting them out. ENV vars, ports, and volumes can all be set/exposed during runtime. And leaving them commented is better than not having any documentation in the essences of time.

Comment (zone117x) [reply Dockerfile]: Done. I left the ENV because a few are required and all of them are good defaults for running without extra config

**Review by CharlieC3 — APPROVED** [review]

**Pull request merged**